### ISSUE TITLE
Search result numeric range min/max wrong types

### ISSUE THREAD

**Issue by chvck** [couchbase/protostellar#36]
They should be float.

**Comment by dnault** [comment]
Or better yet, `double`?

**Comment by dnault** [comment]
These should be optional, right? Since either endpoint could be unbounded.

**Comment by brett19** [comment]
https://github.com/blevesearch/bleve/blob/master/search/facet/facet_builder_numeric.go#L37-L38
We should be using double since the underlying service uses float64.

**Comment by brett19** [comment]
> These should be optional, right? Since either endpoint could be unbounded.

Does that mean both the request _and_ response sides of this should have the min/max being optional?

**Comment by dnault** [comment]
@brett19 Yes, both endpoints should be optional, in both the request and response, for both Numeric and Date ranges. 

If I recall correctly, at least one endpoint must be specified, otherwise the server complains.
